sci-libs/factory/factory-3.1.1-r2.ebuild

```diff
@@ -39,4 +37,12 @@ src_configure() {
 	autotools-utils_src_configure
 }
 
-# TODO: get rid of factories static libs ?
+src_install() {
+	autotools-utils_src_install
+
+	# Passing --disable-static to configure won't disable the build of static libs,
+	# as libtool isn't used. Therefore the static libs are deleted after installed to ${D}.
+	if ! use static-libs; then
+		find "${D}" -type f -name "*.a" -delete || die
```

Comment:
Please use `${ED}`. And it could be shortened to `use !static-libs || find ... || die` if you like.